src/tailscale/tailscaled-entrypoint.sh

```diff
@@ -5,6 +5,43 @@
 
 set -euxo pipefail
 
+USERSPACE_SOCKS_FLAGS=""
+if [[ ! -f /dev/net/tun ]]; then
```

Comment:
I think this will insist that it be a plain file rather than a device, I guess we might want `-e` here.